data-migrations/2024-12-17-1334-create-feedbacks.sql

```diff
@@ -0,0 +1,32 @@
+# Rounds of administrative feedback for a Data Management Plans (DMPs)
+CREATE TABLE `feedbacks` (
```

Comment:
Not a big deal, but I think we can just call this table `feedback`, since `feedbacks` is not a word.

Reply:
Yeah true. haha. I will change it to `feedback`